### DISCUSSION TITLE
PickList: Add an option to set maximum number of items in target list

### DISCUSSION THREAD

**Discussion by PranKe01** [primefaces/community#668] [PrimeNG]
Lets assume we use the PickList to manage users. In some scenarios it would make sense, to add a limit of users for the target list.
Or lets assume we use the PickList to manage a soccer team, where a maximum of 11 players are allowed.

I think an option like [maximumTargetItems]="11" or so would be nice, which disables the "move to right" buttons when that number is reached and also disallows drag&drop and so on.

Same could be possible to define a minimumTargetItems property, to make sure at least one item keeps in list (for example when defining a receipt with ingrediants).

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the feedback, we've added it in our internal backlog.
